sdk/tracing/src/test/java/io/opentelemetry/sdk/trace/TracerSdkTest.java

```diff
@@ -151,8 +151,13 @@ void stressTest_withBatchSpanProcessor() {
           StressTestRunner.Operation.create(2_000, 1, new SimpleSpanOperation(tracer)));
     }
 
+    // Needs to correlate with the BatchSpanProcessor.Builder's default, which is the only thing
+    // this test can guarantee
+    final int defaultMaxQueueSize = 2048;
+
     stressTestBuilder.build().run();
-    assertThat(countingSpanExporter.numberOfSpansExported.get()).isEqualTo(8_000);
+    assertThat(countingSpanExporter.numberOfSpansExported.get())
+        .isGreaterThanOrEqualTo(defaultMaxQueueSize);
```

Comment:
This needed to change given that we are now fully async. We can't guarantee that all spans get through. This is ok though as it is in line with the `BatchSpanProcessor`'s contract.

In reality, this test does see several thousand get through.